config.TEMPLATE.inc.php

```diff
@@ -134,6 +134,11 @@
 ; 0, unvalidated accounts will never be removed. Use this setting to automatically remove bot registrations.
 user_validation_period = 28
 
+; Turn sandbox mode to On in order to prevent the software from interacting with outside systems.
+; Use this for development or testing purposes.
+: Affected services are email, datacite, crossref, orcid and paypal payment
```

Comment:
Probably best not to list affected services -- many of them are in plugins, so we can't list them all. But also, the leading character here is a `:`, which is not a comment.